source/elements/oneDAL/source/bibliography.rst

```diff
@@ -4,8 +4,18 @@
 Bibliography
 ============
 
-For more information about algorithms implemented in |short_name|, refer to the
-following publications:
+For more information about algorithms implemented in |dal_full_name|
+(|dal_short_name|), refer to the following publications:
+
+.. [OpenCLSpec]
+   Khronos OpenCL Working Group, The OpenCL Specification
+   Version:2.1 Document Revision:24
+   Available from `opencl-2.1.pdf <https://www.khronos.org/registry/OpenCL/specs/opencl-2.1.pdf>`_
+
+.. [SYCLSpec]
+   Khronos®OpenCL™ Working Group — SYCL™ subgroup, SYCL™ Specification
+   SYCL™ integrates OpenCL™ devices with modern C++, Version 1.2.1
+   Available from `sycl-1.2.1.pdf <https://www.khronos.org/registry/SYCL/specs/sycl-1.2.1.pdf>`_
 
 .. [Lloyd82]
    Stuart P Lloyd. *Least squares quantization in PCM*. IEEE Transactions on
```

Comment:
@outoftardis, this is for our records, or you probably have the immediate answer. We do bibliography in this style, and I assume this is okay.  However how do the other components do their bibliography?

Reply:
I couldn't find bibliography in other spec components (except for [oneMKL](https://spec.oneapi.com/versions/0.6.0/oneMKL/GUID-732F9EE1-BCEC-4D9B-9B93-AF5499B21140.html), which follows the same style as oneDAL)